### DISCUSSION TITLE
Just a question about compatibility with non docker services

### DISCUSSION THREAD

**Discussion by wieluk** [bubuntux/nordvpn#122] [Q&A]
I have never used docker yet but trying to get the Nordvpn client running on my pi while having pivpn as vpn server made only problems I couldn't solve. So i am looking for a alternative nordvpn client with wireguard. 

My question is about your part of "Local Network access to services connecting to the internet through the VPN."
Is it possible to direct services not in a docker to this docker ? for example my qbittorrent-nox ? or radarr,sonarr?

**Comment by bubuntux** [comment]
In theory is possible, you would need to change the network parameter to host and adjust the interface device inside the start_vpn.sh according to the host device, that is something that i planned to implement a long time ago, but it bring a lot of complications,  honestly is better just having those services inside of containers and inside of the vpn network.

**Reply by charliwest** [reply]
Would this work to proxy devices on my home network out?
